src/components/NavBar/NavBar.tsx

```diff
@@ -1,125 +1,491 @@
     setExpanded(!expanded);
+    setMobileMenuAnchorEl(event.currentTarget);
   };
 
-  const handleClose = () => {
+  const handleMobileMenuClose = () => {
+    setMobileMenuAnchorEl(null);
+
     setExpanded(false);
   };
 
+  const handleLogout = () => {
+    signOutUser();
+    handleMenuClose();
+  };
+
+  const handleMenuOpen = (event: React.MouseEvent<HTMLButtonElement>) => {
+    setAnchorEl(event.currentTarget);
+  };
+
+  const handleMenuClose = () => {
+    setAnchorEl(null);
+  };
+
+  const toggleTheme = () => {
+    SetMode(!mode);
+  };
+
   const userPic = localStorage.getItem("userPic");
   const is_signup = userPic ? true : false;
   const userUid = localStorage.getItem("userUid");
 
   const handleDashboard = () => {
     if (!is_signup) setShow(!show);
-    handleClose();
+    handleMobileMenuClose();
   };
 
+  const mobileMenuItems = (
+    <>
+      <MenuItem
+        component={Link}
+        to="/"
+        onClick={handleMobileMenuClose}
+        sx={{
+          fontWeight: 700,
+          color: "white",
+
+          "&:hover": {
+            backgroundColor: "#A0DEFF",
+            color: "black",
+          },
+        }}
+      >
+        Home
+      </MenuItem>
+      <MenuItem
+        component={Link}
+        to="/dashboard"
+        onClick={() => {
+          handleMobileMenuClose();
+          handleDashboard();
+        }}
+        sx={{
+          fontWeight: 700,
+          color: "white",
+
+          "&:hover": {
+            backgroundColor: "#A0DEFF",
+            color: "black",
+          },
+        }}
+      >
+        Dashboard
+      </MenuItem>
+      <MenuItem
+        component={Link}
+        to="/events"
+        onClick={handleMobileMenuClose}
+        sx={{
+          fontWeight: 700,
+          color: "white",
+
+          "&:hover": {
+            backgroundColor: "#A0DEFF",
+            color: "black",
+          },
+        }}
+      >
+        Events
+      </MenuItem>
+      <MenuItem
+        sx={{
+          fontWeight: 700,
+          color: "white",
+
+          "&:hover": {
+            backgroundColor: "#A0DEFF",
+            color: "black",
+          },
+        }}
+      >
+        <CreateEvent onNavLinkClick={handleMobileMenuClose} />
+      </MenuItem>
+    </>
+  );
+
+  const screenTheme = useTheme();
+  const isSmallScreen = useMediaQuery(screenTheme.breakpoints.down("sm"));
+  const isMediumScreen = useMediaQuery(screenTheme.breakpoints.down("lg"));
+
+  const iconPadding = isSmallScreen ? "10px 12px" : "20px 32px";
+  const mdGap = isMediumScreen ? "16px" : "32px";
+  const mdFontSize = isMediumScreen ? "16px" : theme.fontSize.textBody;
   return (
-    <Navbar
-      bg="dark"
-      variant="dark"
-      expand="lg"
-      expanded={expanded}
-      style={{ userSelect: "none" }}
-      fixed="top"
+    <AppBar
+      position="fixed"
+      sx={{
+        backgroundColor: theme.colors.darkBackground,
```

Comment:
Add this to have the desired functionality:
```typescript
  background: isHome ? "transparent" : theme.colors.darkBackground,
  backdropFilter: isHome ? "blur(12px)" : "none",
```

Reply:
we need to chage the app background color to black or remove the padding of top because in transparent we have while part at the top because of it.

![image](https://github.com/Tanay-ErrorCode/lupo-skill/assets/124615886/0fec7e32-0d49-493e-81f6-c28d2ccf87a0)

we have white part at top
![image](https://github.com/Tanay-ErrorCode/lupo-skill/assets/124615886/db99d256-d069-4745-aca1-3749610258ec)

Reply:
@Utsavladia Its better to change the padding in such a way so that it does not overlap in any page.

Reply:
I am unable to login using signInWithGooglePopup It gives some error in console like "popup.ts:302 Cross-Origin-Opener-Policy policy would block the window.closed call." it would be great if I could get the help setting up.

Reply:
We will have to add padding in each page else it's content will overlap with the Navbar.

Reply:
fixed the padding and it seems as expected now. Thanks for suggestion.
![image](https://github.com/Tanay-ErrorCode/lupo-skill/assets/124615886/c696806f-7927-4248-8378-7cdd8619f4bd)